piker/ui/order_mode.py

```diff
@@ -41,11 +41,30 @@
 
 
 class Position(BaseModel):
+    '''Basic pp representation with attached fills history.
+
+    '''
     symbol: Symbol
     size: float
-    avg_price: float
+    avg_price: float  # TODO: contextual pricing
+    fills: Dict[str, Any] = {}
+
+
+class OrderDialog(BaseModel):
```

Comment:
Ahh right this was the other main client-side addition:  a type to keep track of the order messages for easier analysis by the order mode code.